kitsune/search/tests/test_api.py

```diff
@@ -122,15 +106,15 @@ def test_invalid_numbers(self):
             'q': 'search',
         })
         eq_(res.status_code, 400)
-        eq_(res.data['detail'], {
-            'max_questions': [u'Enter a whole number.'],
-            'max_documents': [u'Enter a whole number.'],
+        eq_(res.data, {
```

Comment:
I don't know. If you tested buddyup, then it's probably fine, but if it isn't, then we have to deal with a lot more work.

I'll check Fjord's sumo suggester, but I don't think I do anything with errors other than ignore them, so I think that code will be fine.

I'm not sure what else uses SUMO APIs.